cengage/react-magma
# feat(treeview): add ability to change items selection outside

Now it's possible to control TreeView state outside of this component by using `apiRef` prop. Available actions: selectItem, selectAll, clearAll.

feat #1286

Issue: #1286

## What I did


## Checklist 

## How to test
Storybook -> TreeView -> Complex
Click on 'Select all' button - all items should be selected
Click on 'Clear all' button - all items should be unselected
I some item was selected then at the bottom appears corresponding tag. If remove this tag - item will be unselected
